lib/sentry/options.ex

```diff
@@ -191,6 +194,10 @@ defmodule Sentry.Options do
 
   @create_event_opts_schema NimbleOptions.new!(@create_event_opts_schema_as_keyword)
 
+  @send_transaction_opts_schema_as_keyword Keyword.merge(@common_opts_schema_as_keyword, [])
+
+  @send_transaction_opts_schema NimbleOptions.new!(@send_transaction_opts_schema_as_keyword)
```

Comment:
We don't need both and the `merge(_, [])` I think. Just go with

```suggestion
  @send_transaction_opts_schema NimbleOptions.new!(@@common_opts_schema_as_keyword)
```